README.md

```diff
@@ -31,12 +31,23 @@ opa {
 
 ## Tasks
 
-The following tasks are currently added by the plugin:
-* `testRego` - Runs `opa test {srcDir} {testDir}` .
+The plugin adds the following tasks:
+* `testRego` - Runs `opa test {srcDir} {testDir}` (see below).
 * `testRegoCoverage` - Runs `opa test {srcDir} {testDir} --coverage` saving report in `build/report/opa` directory. 
 * `startOpa` - Start OPA in background for subsequent tasks like integration tests. 
 * `stopOpa` - Stop OPA process started by `startOpa`.
 
+### testRego
+
+The `testRego` task runs the unit tests found in `testDir` with all policies provided in `srcDir`. If not provided, 
+these directories default to `src/main/rego` (rego policies) and `src/test/rego` (rego tests) respectively. 
+
+#### JUnit XML test results
+
+The `testRego` task automatically converts the OPA test command output into JUnit XML and writes the output to the
+`build/tests-results/opa` directory. This enables any tool or system (such as CI/CD servers) that knows how to parse
```

Comment:
Shouldn't it be `build/test-results/rego`?

Reply:
The "normal" Java one is named `build/test-results/test` which I take it is named after the task (but God knows) and if that's the case I guess `testRego` would be the correct one here. I'm definitely open to changing this but will need to give it some thought.